bazel/README.md

```diff
@@ -1,12 +1,23 @@
 # Bazel
 
-## Alternative Official Image
+The `gcr.io/cloud-builders/bazel` image is maintained by the Cloud Build team,
+but it may not support the most recent features or versions of Bazel. We also do
+not provide historical pinned versions of bazel.
 
-A supported `bazel` image, including multiple tagged versions,
-is available at
-http://gcr.io/cloud-marketplace-containers/google/bazel.
+A supported `bazel` image, including multiple tagged versions, is maintained by
+the Bazel team at http://gcr.io/cloud-marketplace-containers/google/bazel.
+
+To migrate to the Bazel team's official Bazel image, make the following changes
+to your `cloudbuild.yaml`:
+
+```
+- name: 'gcr.io/cloud-builders/bazel'
++ name: 'gcr.io/cloud-marketplace-containers/google/bazel'
```

Comment:
nit: I might be misreading the Git diff, but it looks like the `+` characters were carried over into the paste for the second build step

Reply:
oh never mind, that's intentional. ignore

Reply:
it wasn't immediately clear to me that the `-` meant remove and `+` meant add, but I understand it in context